### PR TITLE
fix(package): add `webpack >= v4.0.0` (`peerDependencies`)

### PR DESCRIPTION
**Pull request by michael-ciniawsky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,24 +50,19 @@
     "webpack-merge": "^4.0.0"
   },
   "engines": {
-    "node": ">= 4.3 < 5.0.0 || >= 5.10"
+    "node": ">= 4.8 < 5.0.0 || >= 5.10"
```
Comment (evilebottnawi) [inline package.json]: Breaking change :disappointed:

File: package.json [hunk 2]
```diff
@@ -50,24 +50,19 @@
...
   },
   "peerDependencies": {
     "node-sass": "^4.0.0",
-    "webpack": "^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0"
+    "webpack": "^2.0.0 || ^3.0.0 || ^4.0.0"
```
Comment (evilebottnawi) [inline package.json]: Also breaking change :disappointed: Let's add just `^4.0.0`

File: package.json (hunk 1 above)
Comment (michael-ciniawsky) [reply package.json]: Why is this a breaking change ? But I can revert it in this PR doesn't matter...

File: package.json (hunk 2 above)
Comment (michael-ciniawsky) [reply package.json]: Why is this a breaking change ? Polluting this prop with `alpha`, `beta`, `rc` versions is really not a good idea and most folks should have updated to the latest `webpack@3.0.0` by now I assumed.

File: package.json (hunk 1 above)
Comment (evilebottnawi) [reply package.json]: @michael-ciniawsky let's do it in next major :+1:

File: package.json (hunk 2 above)
Comment (evilebottnawi) [reply package.json]: @michael-ciniawsky someone can use `rc.0` and it can break build, but i agree with you

File: package.json (hunk 1 above)
Comment (michael-ciniawsky) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (michael-ciniawsky) [reply package.json]: Then it breaks :)

**Pull request merged**